sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/aggregate/regressions.scala

```diff
@@ -0,0 +1,124 @@
+/*
```

Comment:
I'd make the file name clearer: `linearRegression.scala`

Reply:
OK